closure/repositories.bzl

```diff
@@ -66,6 +66,8 @@ def closure_repositories(
     omit_org_ow2_asm_util=False,
     omit_phantomjs=False):
   """Imports dependencies for Closure Rules."""
+  if omit_com_google_protobuf_java:
+    fail("omit_com_google_protobuf_java no longer supported and must be not be passed to closure_repositories()")
```

Comment:
Would be nice if the error message also specified either an alternative or a referenced a public issue for users to go look for answers such as "why?"

Reply:
Please see @jart's comment on the original issue I raised for this warning: #248. I'm not in position to summarize that move in dense couple of words (that would be appropriate for the build tool chain  error message), but I would happily add any suggestions from the project maintainers to extend/adjust the current error message, if this would help get this CL merged.